components/prism-tap.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.tap = {
+	fail: /(    )*not ok[^#\{\n]*/,
+	pass: /(    )*ok[^#\{\n]*/,
+	pragma: /(    )*pragma ([\+-])([a-z]+)/,
+	bailout: /(    )*bail out!(.*)/i,
+	version: /(    )*TAP version ([0-9]+)/i,
+	plan: /(    )*([0-9]+)\.\.([0-9]+)( +#[^\n]*)?/m,
```

Comment:
You can replace all occurrences of `([0-9]+)` with `\d+` (no parentheses needed here) in those two regexps.
Also, it seems the `m` could be removed, since the regexp does not use `^` or `$` assertions.
Finally, the `[^\n]*` part could probably be replaced with `.*`.